locale/cups_de.po

```diff
@@ -4443,22 +4443,22 @@ msgid "US Legal"
 msgstr "US Lang"
 
 msgid "US Legal Oversize"
-msgstr ""
+msgstr "US Legal übergross"
```

Comment:
should be "übergroß" not "übergross" if this is not considered to be Swiss german